SQL/0000-00-00-schema.sql

```diff
@@ -711,7 +711,7 @@ CREATE TABLE `mri_upload` (
   `UploadLocation` varchar(255) NOT NULL DEFAULT '',
   `DecompressedLocation` varchar(255) NOT NULL DEFAULT '',
   `InsertionComplete` tinyint(1) NOT NULL DEFAULT '0',
-  `Inserting` tinyint(1) NOT NULL DEFAULT '0',
+  `Inserting` tinyint(1) NOT NULL DEFAULT NULL,
```

Comment:
is it normal to have NOT NULL then default NULL?

Reply:
Very wrong actually!  :) I corrected this in another commit (see below)